@navikt/core/react/src/util/hooks/useTimeout.ts

```diff
@@ -0,0 +1,56 @@
+"use client";
+
+import { useEffect } from "react";
+import { useRefWithInit } from "./useRefWithInit";
+
+type TimeoutId = number;
+
+const EMPTY = 0 as TimeoutId;
+
+class Timeout {
+  static create() {
+    return new Timeout();
+  }
+
+  currentId: TimeoutId = EMPTY;
+
+  /**
+   * Executes `fn` after `delay`, clearing any previously scheduled call.
+   */
+  start(delay: number, fn: () => void) {
+    this.clear();
+    this.currentId = setTimeout(() => {
+      this.currentId = EMPTY;
+      fn();
+    }, delay) as unknown as number; /* Node.js types are enabled in development */
+  }
+
+  isStarted() {
+    return this.currentId !== EMPTY;
+  }
+
+  clear = () => {
+    if (this.currentId !== EMPTY) {
+      clearTimeout(this.currentId as TimeoutId);
```

Comment:
`currentId` is already typed to `TimeoutId`

```suggestion
      clearTimeout(this.currentId);
```

Reply:
Fixed 👍